Expensify/App
# Remove default payment method for paypal

### Details

### Fixed Issues

$ https://github.com/Expensify/App/issues/7518

### Tests

1. Goto settings -> Payments -> Paypal
2. Click on a already added paypal method
3. Verify that there is no set default payment button for it

### QA Steps

1. Goto settings -> Payments -> Paypal
2. Click on a already added paypal method
3. Verify that there is no set default payment button for it
4.  Verify that other payment method can be selected as default method

### Tested On


### Screenshots


#### Web

<img width="1440" alt="web" src="https://user-images.githubusercontent.com/77761491/153881757-46a0d815-e687-4eae-8e0c-8935903dd733.png">

#### Mobile Web

![mweb](https://user-images.githubusercontent.com/77761491/153881075-8f1c31e5-2e61-4caf-b766-4909fdac1336.png)

#### Desktop

<img width="1440" alt="desktop" src="https://user-images.githubusercontent.com/77761491/153880930-c4293cd3-8df3-41b3-87e8-99b6065297e7.png">

#### iOS

![ios](https://user-images.githubusercontent.com/77761491/153880803-86c8f682-eeff-4888-80f1-7d8a6c63e249.png)

#### Android

![android](https://user-images.githubusercontent.com/77761491/153880698-0dcd76fb-e5e2-4987-a2b8-ec237a6d83cb.png)
